files/file.go

```diff
@@ -261,9 +262,13 @@ func (i *FileInfo) readListing(checker rules.Checker) error {
 		} else {
 			listing.NumFiles++
 
-			err := file.detectType(true, false)
-			if err != nil {
-				return err
+			if detectDisabled {
+				file.Type = "blob"
+			} else {
+				err := file.detectType(true, false)
+				if err != nil {
+					return err
+				}
```

Comment:
We should only disable file detection by header. Simple ext based detection shouldn't be disabled.

Reply:
Now I incorporate the changes from [pr](https://github.com/filebrowser/filebrowser/pull/1168).